skbio/metadata/_interval.py

```diff
@@ -30,6 +31,44 @@ def __init__(self, features=None):
                     self.intervals.add(start, end, k)
             self.features = features
 
+    def complement(self, length):
+        """ Reverse complements IntervalMetadata object.
```

Comment:
is `complement` appropriate? better to follow the name of `DNA.reverse_complement`?